src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -411,62 +411,35 @@ internal virtual NamedTypeSymbol GetNativeIntegerType(NamedTypeSymbol underlying
             throw ExceptionUtilities.Unreachable();
         }
 
-        /// <summary>
-        /// Figure out if the target runtime supports default interface implementation.
-        /// </summary>
+        bool IAssemblySymbolInternal.IsStaticClass(INamedTypeSymbolInternal namedType)
```

Comment:
nit: Why is this method a private method implementation, but the other similar ones are not?